CONTRIBUTING.md

```diff
@@ -88,3 +88,58 @@ Every bug fix should be accompanied with a unit test, so that we can prevent reg
 ### Documentation, typos, ...
 
 They are mostly welcome!
+
+## Operator SDK
+
+Coming soon, but for the moment:
+
+Build the operator
+```
+BUNDLE_VERSION=0.16.0
```

Comment:
Isn't this 0.17.1 now?

Reply:
The PR for 0.17.1 hadn't been merged, but it is now. Will change this PR to reflect that.